### ISSUE TITLE
tty doesn’t display keyboard input after exiting Evans

### ISSUE THREAD

**Issue by danielyli** [ktr0731/evans#62]
After quitting Evans, keyboard input fails to display on the screen. This is on Ubuntu 16.04.4 LTS (GNU/Linux 4.4.0-1057-aws x86_64).

To reproduce this problem:

```
$ curl -OL https://github.com/ktr0731/evans/releases/download/0.4.1/evans_linux_amd64.tar.gz
$ tar -xf evans_linux_amd64.tar.gz
$ ./evans --host localhost --port 50052 --package myservice --service MyService --path protos protos/myservice/MyService.proto

new update available:
  current version: 0.4.1
   latest version: 0.4.1
? update? No
? update? (y/N) n
  ______
 |  ____|
 | |__    __   __   __ _   _ __    ___
 |  __|   \ \ / /  / _. | | '_ \  / __|
 | |____   \ V /  | (_| | | | | | \__ \
 |______|   \_/    \__,_| |_| |_| |___/

 more expressive universal gRPC client


myservice.MyService@127.0.0.1:50051> exit
myservice.MyService@127.0.0.1:50051> Good Bye :)

$ <-- cursor is now here
```

Now, when I type into the terminal, I don’t see output.

Only when I run `$ stty sane` to reset all tty settings does it fix the problem.

**Comment by danielyli** [comment]
I also notice that, though I specified `--port 50052`, Evans uses port `50051`. (See output above.)

**Comment by ktr0731** [comment]
> I also notice that, though I specified --port 50052, Evans uses port 50051. (See output above.)

I fixed it in #63.
but the tty problem is under investigation.

**Comment by ktr0731** [comment]
it seems like a bug of c-bata/go-prompt.  
`exit` and `quit` commands which are executed from REPL terminate Evans with `os.Exit`.  
so, maybe deferred functions for cleanup have been not worked.

the alternative method to exit Evans is entering <kbd>C-d</kbd>, it just works.  
I'll find the bug as soon as possible.

**Comment by ktr0731** [comment]
fixed at #66
